doctr/models/zoo.py

```diff
@@ -47,6 +49,7 @@ def ocr_predictor(
     pretrained: bool = False,
     assume_straight_pages: bool = True,
     export_as_straight_boxes: bool = False,
+    preserve_aspect_ratio: bool = False,
```

Comment:
Either we add this one as kwargs or we update the docstring :)

Reply:
I think we can't really add it as a kwarg because kwargs go to the OCRPredictor and we need to instantiate the detection predictor with that

Reply:
Yes but it will be caught by the signature of OCRPredictor. But in this case, I think we should keep this and only update the docstring by adding the arg description